### DISCUSSION TITLE
Need to include WebView2 runtime in compiled windows executable

### DISCUSSION THREAD

**Discussion by HGS-ryanbloom** [tauri-apps/wry#641] [Q&A]
Hello, I need to be able to ship a single standalone executable file, but there are instances where a user might not have WebView2 installed on their windows system (because it might be too old, too bare, etc). I'm unable to use the installer which wix provides, and must instead use just the executable. Is there any way where I can bundle the WebView2 Runtime in the executable? Would it work as a sidecar? [Here](https://docs.microsoft.com/en-us/microsoft-edge/webview2/how-to/static) there is an article on distributing a WebView2 app as a single binary but I'm wondering if this can be done with a tauri app as well. Thanks for any help!

**Comment by FabianLars** [comment]
See https://github.com/tauri-apps/tauri/discussions/4774

Please refrain from posting the same question across multiple repos.
